Tools/cases_generator/generate_cases.py

```diff
@@ -1045,23 +1072,25 @@ def write_metadata(self) -> None:
             self.out.emit("};")
             self.out.emit("#endif")
 
-    def write_metadata_for_inst(self, instr: Instruction) -> None:
-        """Write metadata for a single instruction."""
+    def emit_metadata_entry(self, name: str, fmt: str, flags: int) -> None:
+        flags_strs = [f"{name}{INSTR_FLAG_SUFFIX}"
+                      for i, name in enumerate(INSTRUCTION_FLAGS) if (flags & (1<<i))]
+        flags_s = "0" if not flags_strs else '|'.join(flags_strs)
```

Comment:
Somehow a space around the '|' looks better?
```suggestion
        flags_s = "0" if not flags_strs else ' | '.join(flags_strs)
```